docs/abi-spec.rst

```diff
@@ -419,9 +419,9 @@ In effect, a log entry using this ABI is described as:
 - ``address``: the address of the contract (intrinsically provided by Ethereum);
 - ``topics[0]``: ``keccak(EVENT_NAME+"("+EVENT_ARGS.map(canonical_type_of).join(",")+")")`` (``canonical_type_of`` is a function that simply returns the canonical type of a given argument, e.g. for ``uint indexed foo``, it would return ``uint256``). If the event is declared as ``anonymous`` the ``topics[0]`` is not generated;
 - ``topics[n]``: ``EVENT_INDEXED_ARGS[n - 1]`` (``EVENT_INDEXED_ARGS`` is the series of ``EVENT_ARGS`` that are indexed);
-- ``data``: ``abi_serialise(EVENT_NON_INDEXED_ARGS)`` (``EVENT_NON_INDEXED_ARGS`` is the series of ``EVENT_ARGS`` that are not indexed, ``abi_serialise`` is the ABI serialisation function used for returning a series of typed values from a function, as described above).
+- ``data``: ``abi.encode(EVENT_NON_INDEXED_ARGS)`` (``EVENT_NON_INDEXED_ARGS`` is the series of ``EVENT_ARGS`` that are not indexed, ``abi.encode`` is the ABI encoding function used for returning a series of typed values from a function, as described above).
```

Comment:
Can `abi.encode` here be confused with the language feature? The original description was not specific to the language.

Reply:
Why confused? It is exactly the thing that is used.

Reply:
What could cause confusion is that `EVENT_NON_INDEXED_ARGS` is actually a series of values, i.e. `abi.encode` here can get multiple arguments instead of just a single one.

Reply:
Not even sure this section should mention functions (especially not Solidity functions), but rather explain what's going on?

Reply:
I thought using `abi.encode` would remove some confusion, but I can rewrite it without using solidity functions.

Reply:
Actually wait a second - this is about events, a solidity feature - shouldn't it be fine to reference solidity functions here?

Reply:
ok, perhaps not...